### ISSUE TITLE
Pausing: timer skips seconds or goes into the negative

### ISSUE THREAD

**Issue by chriscz** [DaniGuardiola/TimeDoser#7]
When I click the small (left-hand corner) pause / start button, it does not pause the timer, but instead makes it skip 2 seconds at a time. This seems to happen only when the start button is visible instead of the expected pause button. See https://img42.com/GcdZU. 

Besides the skipping, I've also found that sometimes the time runs into the negative.

I'm not sure how to reproduce it at the moment. But I'll post back if I find out.
Forgot to mention, I'm using version  0.0.9.4.
##

**Comment by DaniGuardiola** [comment]
Hey, does this still happen?

**Comment by chriscz** [comment]
I'm installing the dev version now, will keep you posted. Also, I noted that the version has not been bumped?

**Comment by chriscz** [comment]
I am still on version 0.0.9.4
tldr; the START/PAUSE button in the corner isn't reset when a new timer starts

Here is the scenario that reproduces it:
1. Open timedoser, clean start
2. Let the WORK timer run out, till the BREAK timer starts
3. Now PAUSE the BREAK timer
4. Now SKIP the break
5. The START/PAUSE button will now be in START mode,
   not PAUSE mode as it should be.
6. Clicking the START button will result in double the time running out.

FIX: reset the START/PAUSE button when clicking SKIP

**Comment by DaniGuardiola** [comment]
@chriscz TimeDoser v2 is being retired, so I'm closing all issues. v3 is in the works, and it should replace v2 once I'm done. Thanks for your interest!
